archive/n/nim/fibonacci.nim

```diff
@@ -0,0 +1,19 @@
+# Fibonacci Sample Program in Nim
+import strutils
+
+echo "Input the number of iterations (n): "
```

Comment:
For this program, I'd take a peek at the requirements: https://sample-programs.therenegadecoder.com/projects/fibonacci/

We typically don't prompt for input. Instead, we pull it directly off the command line. Does Nim support command line args?

Reply:
Yeah once I saw the build failed I started looking into that. There are a number of ways, just trying to figure out an os-agnostic method.